src/test/groovy/com/intuit/graphql/orchestrator/integration/XtextConflictResolverSpec.groovy

```diff
@@ -92,4 +99,234 @@ class XtextConflictResolverSpec extends BaseIntegrationTestSpecification {
         queryType.getFieldDefinition("bar") != null
         queryType.getFieldDefinition("foo").type == queryType.getFieldDefinition("bar").type
     }
+
+    def "Query Service A"() {
+        given:
+        SimpleMockServiceProvider serviceA = createMockService("SVC_A",
```

Comment:
Should we use inline schemas @CNAChino @kmoore-intuit ?

Reply:
The schemas are not too long and having it in localized places would enhance readability.

Reply:
lets do inline schemas if that is a possibility

Reply:
Would it be also okay to inline the GraphQL queries?

Reply:
I'm leaning on short schemas and short queries and have them all in the groovy file

Reply:
We could have the schema queries/response in a separate file for each spec and just import the variable in.  Would help keep the spec files clean and help with consistency.

Reply:
Would it be possible to meet to come to an agreement?

Reply:
Per discussion, the schemas and responses have been incorporated into respective unit test cases.